helm-chart/splunk-kubernetes-logging/values.yaml

```diff
@@ -359,3 +359,21 @@ custom_metadata:
 #     body: jq '.record.cluster_name = "my_awesome_cluster" | .record'
 # ```
 customFilters: {}
+
+
+# `indexFields` defines the fields from the fluentd record to be indexed.
+#
+# The scheme to define an indexed field is:
+#
+# ```
+# ["field_1", "field_2"]
+# ```
```

Comment:
cleanup language here

Reply:
"basically an srray" -> This is a comma separated list of fields

Reply:
include hyperlink to HEC indexed fields info - http://dev.splunk.com/view/event-collector/SP-CAAAFB6